src/commands/guess.ts

```diff
@@ -0,0 +1,179 @@
+import dayjs from 'dayjs';
+import { CommandInteraction, MessageActionRow, MessageButton, PermissionString } from 'discord.js';
+import { RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-types/v10';
+import { Command } from '../interfaces/Command';
+import { t } from '../utils/replyHelper';
+import { getUserStatus, registerUser, getDay, getWord, setPlayed } from '../database';
+import { runAtMidnight } from '../utils/runner';
+import { awaitMessage } from '../utils/msgCollector';
+import { isValid } from '../utils/checkWord';
+import { toDefault, toEmoji } from '../utils/converters';
+import { platform } from '../utils/platform';
+import { check, square } from '../utils/emotes.json';
+
+let usersTries: Record<string, { id: string, attempts: string[] }> = {};
+let activeGames: string[] = [];
+
+runAtMidnight(() => {
+	usersTries = {};
+	activeGames = [];
+});
+
+export default class Guess implements Command {
+	commandStructure: RESTPostAPIChatInputApplicationCommandsJSONBody = {
+		'name': 'adivinhar',
+		'description': 'Tente adivinhar a palavra do dia!',
+	};
+
+	dev = true;
```

Comment:
Almost passed through this one
```suggestion
	dev = false;
```